src/App.js

```diff
@@ -1,24 +1,32 @@
 import React from 'react';
+import { ToastContainer } from 'react-toastify';
+import dotenv from 'dotenv';
+import 'react-toastify/dist/ReactToastify.css';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import { Provider } from 'react-redux';
+import axios from 'axios';
 import UserSignup from './containers/user/signup/signup';
 import store from './redux/store';
 import Signout from './containers/user/signout/signout';
 import Notfound from './NotFound';
-import Demo from './components/Demo';
 import './assets/css/App.scss';
+import Profile from './components/Profile';
+import Navbar from './components/Navbar';
 
+dotenv.config();
+axios.defaults.baseURL = 'https://team-odd-bn-backend-staging.herokuapp.com/api/v1/users';
 const App = () => (
   <Provider store={store}>
     <Router>
       <div className="container">
         <Switch>
-          <Route path="/" exact component={Demo} />
           <Route path="/signup" exact component={UserSignup} />
           <Route path="/signout" exact component={Signout} />
+          <Route path="/profile" exact component={Profile} />
```

Comment:
Could you please first check if the user is logged in? You can achieve this by modifying your route (if token is in the props)like this:
```js
<Route
	exact
	path='/profile'
	render={props => (token ? <profile /> : <Redirect to='/login' />)}
/>
```